### PR TITLE
The solution of the color pixel problem

### PR DESCRIPTION
**Pull request by anahittum**

### CODE REVIEW

File: src/homeworks/week4/color/pixel/ColorPixel.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package homeworks.week4.color.pixel;
+
+public class ColorPixel extends Pixel {
+
+    String color;
```
Comment (gevorgyanarman) [inline src/homeworks/week4/color/pixel/ColorPixel.java:5]: the filed should be private

File: src/homeworks/week4/color/pixel/ColorPixel.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public String toString() {
+        return "ColorPixel{" +
+                "color='" + color + '\'' +
+                ", x=" + x +
```
Comment (gevorgyanarman) [inline src/homeworks/week4/color/pixel/ColorPixel.java:19]: here you will have a problem if you make x and y private. but you should do

File: src/homeworks/week4/color/pixel/Pixel.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package homeworks.week4.color.pixel;
+
+public class Pixel {
+    int x;
```
Comment (gevorgyanarman) [inline src/homeworks/week4/color/pixel/Pixel.java:4]: please make private

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

File: src/homeworks/week4/color/pixel/ColorPixel.java (hunk 2 above)
Comment (anahittum) [reply src/homeworks/week4/color/pixel/ColorPixel.java:19]: @gevorgyanarman can you advise how this problem can be handled?

Comment (anahittum) [reply src/homeworks/week4/color/pixel/ColorPixel.java:19]: Decided to access the fields via getters, what do you think?

**Pull request merged**